### PR TITLE
Expiring sandbox announcement

### PR DESCRIPTION
**Pull request by gboone**
We'll also send this text to all current users by Mailchimp.

### CODE REVIEW

File: content/updates/starting-now-sandboxes-expire-after-90-days.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
++++
+title: "Starting now, Sandboxes expire after 90 days"
+date: 2017-12-06
++++
+
+In our monthly update from November, we mentioned we would start expiring sandboxes after 90 days. Today, we're starting that clock. Here's what to expect.
+ 
+
+* Any sandboxes applications you have right now will be deleted 90 days from today.
```
Comment (jmcarp) [inline content/updates/starting-now-sandboxes-expire-after-90-days.md]: Could we consolidate the first two bullet points?

```
* All sandbox applications and services will be deleted 90 days after you create your first application or service.
```

File: content/updates/starting-now-sandboxes-expire-after-90-days.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+* The apps themselves will be deleted and any managed services attached to them will be disconnected.
+* You will not lose your cloud.gov account.
+* Starting five days before we delete your sandbox, we'll send you an email reminder every day unless you delete your sandbox yourself
+
+If you have an application in your sandbox that you need to keep online for longer than 90 days, you should consider purchasing a paid cloud.gov account. Any prototyping or production account will allow you to host apps for any amount of time. Contact us to learn how to get started with a paid plan.
```
Comment (jmcarp) [inline content/updates/starting-now-sandboxes-expire-after-90-days.md]: Or users can redeploy their applications after their sandbox is cleared. I think we should try to be clearer on this point: users can continue to use their sandbox after we clear its contents--they'll just have to deal with the inconvenience of losing their applications and services every 90 days.

**Comment by jmcarp** [comment]
It might be easier to literally copy and paste text from https://github.com/18F/cg-sandbox/blob/master/templates/notify.tmpl, then revise as necessary to make it sound more like an announcement.

**Comment by pburkholder** [comment]
Here's my attempt to pull in the words from the template, and keep the announcement feel.

```
+++ title: "Starting now, Sandboxes expire after 90 days" date: 2017-12-06 +++

In our monthly update from November, we mentioned we would start expiring sandboxes after 90 days. Today, we're starting that clock. Here's what to expect.

   - Any sandboxes applications you have right now will be deleted 90 days from today, along with service instances, routes, etc., in the sandbox.
  - You will not lose your cloud.gov account.
  - You can start a new 90-day evaluation period just by creating a new app or service.
  - Starting five days before we delete your sandbox, we'll send you an email reminder every day unless you delete your sandbox yourself.

If you have an application in your sandbox that you need to keep online for longer than 90 days, you should consider purchasing a paid cloud.gov account. Any prototyping or production account will allow you to host apps for any amount of time. Contact us to learn how to get started with a paid plan.
```

**Comment by mogul** [comment]
That sounds pretty good to me, though `Sandboxes` shouldn't be capitalized in the subject line.

File: content/updates/starting-now-sandboxes-expire-after-90-days.md (hunk 2 above)
Comment (gboone) [reply content/updates/starting-now-sandboxes-expire-after-90-days.md]: Do we want to advertise that, though? Seems like we'd rather push people toward buying cloud.gov if they need to keep their apps running persistently?

Comment (jmcarp) [reply content/updates/starting-now-sandboxes-expire-after-90-days.md]: I think the text that @pburkholder pulled from existing docs addresses this issue pretty well. Can we check in that text, fix the casing nit @mogul pointed out, and merge?

**Comment by gboone** [comment]
Alright, made some updates based on the feedback here. Let me know if it looks good and we can get this merged!

**Comment by jmcarp** [comment]
This LGTM. I'd like to get confirmation on the announcement date from @brittag or @mogul, then :shipit: .

**Comment by brittag** [comment]
This looks good to me! I ran it locally and added commits with the following changes:

- Remove duplicate post file
- Add link to monthly update from November
- Add link to reference page about sandboxes in general
- More consistent phrasing for "spaces" and "applications"; other minor copyedits (fixed capitalization of sandbox in the title)
- `relref` links instead of raw links, for easier local testing

**Review by mogul — APPROVED** [review]

**Pull request merged**